src/groupeddataframe/splitapplycombine.jl

```diff
@@ -199,12 +199,12 @@ accurate than the standard [`sum`](@ref) function (which uses pairwise summation
 `col => x -> sum(x)` to avoid the optimized method and use the slower, more accurate one.
 
 Column names are automatically generated when necessary using the names `x1`, `x2` and so on
-if `f` is a `Function` or an `AbstractMatrix` is returned by it
+if `f` is a `Base.Callable` or an `AbstractMatrix` is returned by it
```

Comment:
Probably better for users to say "function or type". Same for the signature.

I even wonder whether it's worth allowing types, as the result will be copied to a `DataFrame` anyway. These functions are complex enough.

Reply:
we allow types in `select`. I have updated the docstrings so that this is not a problem (I will push it shortly, but I have 14 small changes to coordinate so it takes time :snail:)